Automattic/wp-calypso
# o2 Blocks: Move to packages/

#### Changes proposed in this Pull Request

- Delete `trial` and `trials` blocks, per p1554183561017100-slack-devex
- Delete `hello-dolly` block (unused)
- Move remaining blocks from `client/gutenberg/extensions` to `packages/o2-blocks`
- Add `package.json`, `webpack.config.js`

#### Testing instructions

`npm run distclean && npm ci`

Verify that blocks are built in `packages/o2-blocks/dist`.

#### Next steps (follow-up PRs)

- see https://github.com/Automattic/wp-calypso/pull/31927/files#r271958662
- Add a `babel.config.js` to move specific transforms from `@automattic/calypso-build`'s
